src/WebView.ios.tsx

```diff
@@ -214,4 +215,4 @@ const isFileUploadSupported: () => Promise<boolean>
 
 const WebView = Object.assign(WebViewComponent, {isFileUploadSupported});
 
-export default WebView;
+export default WebView;
```

Comment:
weird... I don't see what's different here..

Reply:
I guess it's a trailing CR